int32 index
uint64 time_stamp
uint8[20971520] data

uint32 MAX_SIZE = 20971520
